pagerduty_nagios.pl

```diff
@@ -107,6 +167,85 @@ =head1 OPTIONS
 my $opt_verbose;
 my $opt_proxy;
 
+#** @method public is_variable_allowed ($variable_name, $variable_value)
+# Given an variable name, see if we want to allow it to be put in a PagerDuty payload.
+# Returns 1 if it should be carried over, 0 otherwise
+#*
+sub is_variable_allowed {
+	my ($variable_name, $variable_value) = @_;
```

Comment:
for fellow reviewers who don't remember their perl:
> Within a subroutine the array @_ contains the parameters passed to that subroutine. Inside a subroutine, @_ is the default array for the array operators push, pop, shift, and unshift.

Thank you Chris for not using bare array operators in this btw

Reply:
it is 2019, i have the bytes to spare to make this actually readable